lib/Doctrine/Migrations/OutputWriter.php

```diff
@@ -12,10 +12,15 @@ class OutputWriter
     /** @var callable */
     private $callback;
 
+    /** @var string */
+    private $lastMessage = '';
+
     public function __construct(?callable $callback = null)
     {
         if ($callback === null) {
-            $callback = static function ($message) : void {
+
+            $callback = static function (string $message) : void {
```

Comment:
Static closures don't get any object bound to `$this`, we should then change this to:

```suggestion
            $callback = function (string $message) : void {
```